### ISSUE TITLE
Create basic inventory system and items.

### ISSUE THREAD

**Issue by wesker-albert** [wesker-albert/cibo#13]
Create basic player inventory system, and items models.

Items should have the following optional functionality:

* _Stationary_ - not able to be picked up or moved by the player
* _Pickup_ - able to be picked up and dropped by the player

Pickup items should persist in the player inventory when acquired, and disappear from the inventory when dropped. When dropped the item should remain in the current room it was dropped within, available again for pickup.

~~World items respawn to previous set positions on world restart.~~

**Comment by wesker-albert** [comment]
I'm making the decision to omit the following from the AC:

* World items respawn to previous set positions on world restart.

I believe world item spawns need more thought. As I began working on this feature, I realized that I really like the idea of item persistence and degradation. For example:

* If you drop a gun, it will remain where you dropped it until another player or NPC picks it up.
* The gun will have a degradation threshold, and if it reaches said threshold, it will be deleted from the world.

This needs more thought, because I will need to determine was happens at that point. Does the item respawn to it's original location upon degradation? Does it respawn sooner, to avoid scarcity?

TLDR: We'll wait to address the topic of item spawn and deleting.
